src/components/Layout/FeatureItem/SpecialCountdown.js

```diff
@@ -0,0 +1,68 @@
+import React,{useRef, useReducer, useState,useEffect} from 'react'
+import shoeImg from '../../../img/shoes.jpg'
+
+const SpecialCountdown = () => {
+    const [timerDays, setTimerDays]=useState('00');
+    const [timerHours, setTimerHours]=useState('00');
+    const [timerMinutes, setTimerMinutes]=useState('00');
+    const [timerSeconds, setTimerSrconds]=useState('00');
+  
+    let interval = useRef();
```

Comment:
Have you tried this without useRef? I think that would also work by storing the interval timer id in a normal variable and clearing it.